roles/ec2userdata/tasks/main.yml

```diff
@@ -121,6 +121,12 @@
     dest: /usr/local/bin
     remote_src: true
 
+- name: Install Podman
```

Comment:
Since this is packaged, this should just be added to the list at L15.

Reply:
Makes sense.  done.